.github/workflows/PullRequestHandler.yaml

```diff
@@ -41,11 +41,13 @@ jobs:
           Add-Content -Path $env:GITHUB_OUTPUT -Value "prfolder=$prfolder"
           $prPath = Join-Path $location $prFolder
           New-Item -Path $prPath -ItemType Directory | Out-Null
+          $prfilesChanged = @()
```

Comment:
```suggestion
          $prFilesChanged = @()
```

Reply:
Again, this needs to be fixed in AL-Go itself :)

Reply:
Yeah - we can fix this in AL-Go